services/workflows-service/src/common/utils/random/index.ts

```diff
@@ -0,0 +1,35 @@
+const SYMBOLS = '!@#$%^&*()_+-=[]{}|;:,.<>?';
```

Comment:
what is the usage of this? what does it have to do with the PR?
overall why?

Reply:
its not used here, just added a utils function that has the option to generate a random string with symbols.